### PR TITLE
update docs/reference/commandline/service_create.md

### PR DESCRIPTION
**Pull request by AkihiroSuda**
I think this PR improves the readability a little bit.

Signed-off-by: Akihiro Suda suda.akihiro@lab.ntt.co.jp

### CODE REVIEW

**Comment by SvenDowideit** [comment]
LGTM

**Comment by AkihiroSuda** [comment]
Thank you for reviewing, but I think the milestone is 1.12.**2**?

**Comment by vdemeester** [comment]
LGTM 🐸

File: docs/reference/commandline/service_create.md [hunk 1]
```diff
@@ -139,9 +139,9 @@ metadata](../../userguide/labels-custom-metadata.md).
 
 ### Set service mode
 
-Is this a replicated service or a global service. A replicated service runs as
-many tasks as specified, while a global service runs on each active node in the
-swarm.
+You can set the service mode to "replicated" (default) or to "global". A 
+replicated  service runs  as many tasks as specified, while a global service 
+runs on each  active node in the swarm.
```
Comment (mstanleyjones) [inline docs/reference/commandline/service_create.md]: nit: extra space

**Pull request merged**